src/main/java/com/rk/rentacarapp/services/VehicleService.java

```diff
@@ -0,0 +1,26 @@
+package com.rk.rentacarapp.services;
+
+import com.rk.rentacarapp.model.Vehicle;
+import com.rk.rentacarapp.model.VehicleType;
+
+import java.util.List;
+
+public interface VehicleService {
+
+    List<Vehicle> findAllVehicles();
+
+    Vehicle findVehicleById(Long id);
```

Comment:
Would there be a difference if you're using long vs Long? If there is a specific reason you're using Long than it's fine otherwise I'd always use a simple data type